packages/triejs/LICENSE

```diff
@@ -1,20 +0,0 @@
-Copyright (C) 2012 Paul Thurlow
```

Comment:
Not sure if we can re-license this that easily, maybe let's just leave it as is for now until we drop this dependency completely and remove this from the repo?